YvesIraguha/TeaHouse
# #169725565 Enable CRUD Functionalities on Individual Pieces

#### What does this PR do?
- CRUD functionalities on individual pieces
#### Description of Task to be completed?
- Create/read/delete/update pieces
- Protect create/delete/update for admin only
- Test added functionalities
#### How should this be manually tested?
-  Clone the repo and checkout the branch `crud-individual-pieces-169725565`
-  Run `npm i` to install all dependencies
-  Ensure that you have all environment variables specified in this file `.env.sample`
-  Run `npm run test` to run tests, they should pass. 
-  Run `npm run start` to start the server.
-  Use client tool, preferably postman, and consume `/api/v1/individual-pieces` endpoints with correct required fields, you should see the outcome. 
#### What are the relevant pivotal tracker stories?
 #[169725565](https://www.pivotaltracker.com/story/show/169725565)
